src/org/stringtemplate/v4/Interpreter.java

```diff
@@ -451,7 +451,7 @@ protected int _exec(STWriter out, InstanceScope scope) {
                     strIndex = getShort(code, ip);
                     ip += Bytecode.OPND_SIZE_IN_BYTES;
                     o = self.impl.strings[strIndex];
-                    n1 = writeObjectNoOptions(out, scope, o);
+                    n1 = writeText(out, scope, (String)o);
```

Comment:
This cast could be eliminated by passing `self.impl.strings[strIndex]` directly to `writeText()` or by using a separate `String` variable instead of `Object o`. I didn't do either because to me, both variants didn't seem to fit the local code style.